src/go/rpk/pkg/cli/cmd/cluster/config/lint.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	"gopkg.in/yaml.v3"
+)
+
+func newLintCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lint",
+		Short: "Remove any deprecated content from redpanda.yaml.",
+		Long: `Remove any deprecated content from redpanda.yaml.
+
+Deprecated content includes properties which were set via redpanda.yaml
+in earlier versions of redpanda, but are now managed via Redpanda's
+central configuration store (and via 'rpk cluster config edit').
+
+This command is not for general changes to cluster configuration: use this only
+when redpanda will not start due to a configuration issue.
+
+This command erases a named property from the local node's internal 
```

Comment:
Is this in addition to updating the redpanda.yaml file? Is the local node internal cache something within redpanda itself? If so, I'm not sure where to describe it, but some sentence somewhere (here? here & reset? maybe just in `rpk cluster config` itself?) may be beneficial.

Reply:
This was the help text that was wrongly copied from reset.go, but the comment is pertinent.

I've tweaked the wording to say "from _an_ internal cache of the cluster configuration" so that it is clearer that user isn't supposed to know anything about what this cache is, just Trust Me It Exists.  This is the nature of commands that are for use in disaster recovery, touching things that aren't usually for the user to know about.